### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by blocosted**
@sberthier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,93 +1,15 @@
...
-function diskUsage(dir) {
-    return wait(fs.readdir(dir)).reduce((size, name) => {
-        const sub = join(dir, name);
-        const stat = wait(fs.stat(sub));
-        if (stat.isDirectory()) return size + diskUsage(sub);
-        else if (stat.isFile()) return size + stat.size;
-        else return size;
-    }, 0);
-}
+Contrary to `f-promise`, `f-promise-async` does not allow to manage promises, but only provides some tools.
```
Comment (sberthier) [inline README.md]: but only provides some tools help promise manipulation and synchronization.

File: README.md [hunk 2]
```diff
@@ -102,114 +24,93 @@ import { wait } from 'f-promise-async';
...
-myFunnel(() => { /* code with at most 10 concurrent executions */ });
-```
+### run/wait
 
-The `funnel` function can also be used to implement critical sections. Just set funnel's `max` parameter to 1.
+* `promise = run(() => { wait(promise/callback); ... })`  
+  create a fake coroutine to write asynchronous code in a asynchronous way !  
+  keep that methode to ease f-promise conversion to async/await  
+  but also for global context
+  * `promise = run(fn)` create a fake coroutine.
```
Comment (sberthier) [inline README.md:40]: Contrary to legacy f-promise, you can use other tools without `run`, except `Continuation local storage`.

File: README.md [hunk 3]
```diff
@@ -102,114 +24,93 @@ import { wait } from 'f-promise-async';
...
-* `cx = fpromise.context()`  
-  returns the current context.
+### Continuation local storage (CLS)
 
-* `fn = fpromise.withContext(fn, cx)`  
-  wraps a function so that it executes with context `cx` (or a wrapper around current context if `cx` is falsy).
+* `result = await withContext(fn, cx)`  
+  wraps a function so that it executes with context `cx` (or a wrapper around current context if `cx` is falsy).  
   The previous context will be restored when the function returns (or throws).  
   returns the wrapped function.
```
Comment (sberthier) [inline README.md]: It is the only tool that need to be executed in a `run`. Can be a nested function call (coroutine).

**Review by sberthier — COMMENTED** [review]
Quelques remarques mineur.

File: README.md (hunk 1 above)
Comment (blocosted) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (blocosted) [reply README.md:40]: done

File: README.md (hunk 3 above)
Comment (blocosted) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (sberthier) [reply README.md]: Micro correction: `to help`

but only provides some tools to help promise manipulation and synchronization.

File: README.md (hunk 2 above)
Comment (sberthier) [reply README.md:40]: \` autour du `run` stp

File: README.md (hunk 3 above)
Comment (sberthier) [reply README.md]: \` autour du `run` stp

Comment (blocosted) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (blocosted) [reply README.md:40]: done

File: README.md (hunk 1 above)
Comment (blocosted) [reply README.md]: done

**Pull request merged**